src/style.css

```diff
@@ -1 +1,110 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+body {
+  margin-top: 20px;
+}
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). It's better to avoid potential margin collapse issues by sticking to either top or bottom margins for elements. Consider removing the `margin-top` from the `body` and applying it to the first element within the body if necessary.